test/queues.c

```diff
@@ -2,6 +2,36 @@
 #include "greatest.h"
 #include "queues.h"
 
+TEST test_queue_length(void)
+{
+        queues_init();
+
+        struct notification *n;
+
+        n = test_notification("n1", 0);
+        queues_notification_insert(n);
+        queues_notification_close(n, REASON_UNDEF);
+
+        n = test_notification("n2", 0);
+        queues_notification_insert(n);
+        queues_update(STATUS_NORMAL);
+
+        n = test_notification("n3", 0);
+        queues_notification_insert(n);
+
+        QUEUE_LEN_ALL(1,1,1);
+
+        ASSERT_EQm("Queue waiting has to contain an element",
+                   1, queues_length_waiting());
+        ASSERT_EQm("Queue displayed has to contain an element",
+                   1, queues_length_displayed());
+        ASSERT_EQm("Queue history has to contain an element",
+                   1, queues_length_history());
```

Comment:
These are redundant: if `QUEUE_LEN_ALL` fails above they won't get run either way, if it succeeds they will pass.

Reply:
I already wanted to rebase and remove it. The test seems duplicated.

Bit it isn't: The test wants to test `queues_length_*`, so we have those `ASSERT_EQm`.